src/components/Concept/ConceptPage.jsx

```diff
@@ -0,0 +1,268 @@
+/*
```

Comment:
Er det noen grunn til at vi ikke skriver typescript i listing-frontend?

Reply:
Dunno man, i wish we did tho

Reply:
Jeg er med for å kickstarte typescript i både listing-frontend og ndla-frontend

Reply:
+1

Reply:
+2

Reply:
+3